lms/djangoapps/support/static/support/templates/manage_user.underscore

```diff
@@ -0,0 +1,43 @@
+<div class="manage-user-search">
+    <form class="manage-user-form">
+        <label class="sr" for="manage-user-query-input"><%- gettext('Search') %></label>
+        <input
+            id="manage-user-query-input"
+            type="text"
+            name="query"
+            value="<%- user %>"
+            placeholder="<%- gettext('Username') %>">
+        </input>
+        <input type="submit" value="<%- gettext('Search') %>" class="btn-disable-on-submit"></input>
+    </form>
+</div>
+
+<% if (user_profile.get('is_active')) { %>
```

Comment:
Why checking this? what if user profile is inactive?

Reply:
if 'is_active' is True only then user detail will be shown.